texthero/representation.py

```diff
@@ -136,42 +48,38 @@ def count(
     min_df=1,
     max_df=1.0,
     binary=False,
-) -> pd.Series:
+) -> pd.DataFrame:
     """
     Represent a text-based Pandas Series using count.
 
-    Return a Document Representation Series with the
-    number of occurences of a document's words for every
-    document.
+    Rows of the returned DataFrame represent document whereas 
+    columns are terms. The value in the cell document-term is
+    the number of the term in this document. The output is sparse.
     TODO add tutorial link
 
     The input Series should already be tokenized. If not, it will
     be tokenized before count is calculated.
 
-    Use :meth:`hero.representation.flatten` on the output to get
-    a standard Pandas Series with the document vectors
-    in every cell.
-
     Parameters
     ----------
     s : Pandas Series (tokenized)
 
     max_features : int, optional, default to None.
```

Comment:
replace it with `default=None` (to be consistent with the rest), change also in the other functions

Reply:
Will rather do this in a separate PR across all files (see comment below)

Reply:
OK!